troubleshooting/troubleshooting-android-push-notifications-stopped-working.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Push Notifications to Android stopped working
+description: Check your Google API key if you receive the error "Error authenticating to Google Cloud Messaging"
+type: troubleshooting
+page_title:  Unsuccessful authentication to Google Cloud Messaging
+slug: troubleshooting-android-push-notifications-stopped-working
+position:
+tags: Google Cloud Messaging, GCM, Android, push notifications, FCM
+teampulseid: 
+ticketid: 1044096, 1045723
+pitsid:
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Notifications (Backend Services)</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Push Notifications, Android</td>	
+  </tr>
+</table>
+
+## Description
+
+Your Android app may have stopped receiving push notifications even they have been working for a significant amount of time. [Checking the reason for failing push notifications](http://docs.telerik.com/platform/backend-services/javascript/push-notifications/troubleshooting/push-trb-all.html#q-how-do-i-find-out-what-is-the-reason-for-not-receiving-push-notifications) in {{site.tp}} reveals the following error message. 
```

Comment:
I would rather say this to keep the customer context:
"Android app has suddenly stopped receiving push notifications due to the following error `Error authenticating to Google Cloud Messaging server.`."